.github/workflows/scala.yml

```diff
@@ -0,0 +1,21 @@
+name: Scala CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
```

Comment:
Wydaje mi się, że docelowo dobrze byłoby uruchamiać się na linuxie jeśli wgl nie w jakimś dockerze. Pomimo iż to wszystko i tak działa w jvm to sugerowałbym jednak spróbować z linuxem

Reply:
Ok, zmienię na ubuntu 20.04